File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    var answer = '';
    var sd = [];
    for (let i = 1;i<food.length;i++){
        answer += String(i).repeat(Math.floor(food[i]/2));
    }
    answer += '0';
    for (let i = food.length;i>0;i--){
        answer += String(i).repeat(Math.floor(food[i]/2));
    }
    return answer;
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

string solution(vector<int> food) {
    string answer = "";
    string a = "";
    
    for(int i = 1;i<food.size();i++){
        int repeat = food[i];
        for(int j = 0;j<repeat/2;j++){
            a += to_string(i);
        }
    }
    string b = a;
    a += "0";
    reverse(b.begin(),b.end());
    a += b;
    answer = a;
    return answer;
}